content/02.introduction.md

```diff
@@ -155,14 +155,23 @@ Abnormal clotting (thromboinflammation or coagulopathy) has been increasingly di
+
+Many other factors influence risk of serious COVID-19 outcomes.  Genetic factors may play a role in risk of respiratory failure for COVID-19 [@doi:10.1056/NEJMoa2020283; @doi:10.1093/gerona/glaa131; @doi:10.1101/2020.06.16.155101].  Diabetes may increase the risk of lengthy hospitalization [@doi:10.1016/j.cmet.2020.04.021] or of death [@doi:10.1016/j.cmet.2020.04.021;@doi:10.1007/s00592-020-01546-0].  [@doi:10.1111/dom.14057] and [@doi:10.1152/ajpendo.00124.2020] discuss possible ways in which COVID-19 and diabetes may interact.  Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2 [@doi:10.1016/j.metabol.2020.154262; @doi:10.1101/2020.04.23.20076042]. Obesity is considered an underlying risk factor for other health problems, and the mechanism for its contributions to COVID-19 hospitalization or mortality is not yet clear [@doi:10.1016/j.medj.2020.06.005].
+
+Study of disparities and differences among groups in rates of hospitalization or death from COVID-19 are complicated by the fact that risk may be elevated in multiple ways.  An individual may be more likely to contact the virus, more likely to get infected once exposed, be more likely to have serious complications once infected, or be less likely to get adequate help once they are seriously ill. Although these are difficult to disaggregate, they suggest different possible interventions and each deserves consideration.
+The epidemiological characteristics and clinical presentation of COVID-19 may differ somewhat in different areas of the world, presumably due to differential reporting, different age structures, and/or different risk factors [@doi:10.1007/s00592-020-01546-0].  Furthermore, because different subpopulations may have somewhat different vulnerabilities, needs, and resources, we recommend that researchers publishing studies on diagnostics and therapeutics take extra care to be clear about the demographic and medical characteristics of their sample, in order to facilitate discussions of the degree to which results may generalize or differ in other settings.
+
+Besides the direct harms caused by infection, many populations are in indirect risk of serious harm due to the social and economic effects of the pandemic and of the efforts needed to fight it.  These might include 
+individuals with substance use disorder [@doi:10.7326/M20-1141], victims of natural disasters [@doi: 10.1038/s41558-020-0804-2], and victims of human trafficking [@doi: 10.1016/j.eclinm.2020.100409].  The pandemic could also delay the fight against other major infectious diseases, such as HIV, malaria and tuberculosis, potentially leading to a further increase of mortality [@doi:10.25561/78670]. Although they are beyond the scope of this paper, further research is needed in order to prevent these harms.
```

Comment:
Format: There are 3 sentences here in 1 line 174. Separate into 3 lines.

Reply:
Thanks to @rando2 for fixing the spacing!  If even my sentences can't socially distance, then I am indeed in peril... :-)